examples/rockylinux-mofed/README.md

```diff
@@ -0,0 +1,13 @@
+# Rocky Linux with the Mellanox OFED
+
+This container definition demonstrates building a Rocky Linux based container incorporating the Mellanox OFED for InfiniBand support.
+
+This definition requires that the Mellanox OFED downloadable for the desired version be placed in the `rockylinux-mofed` directory.
+Download the `.tgz` version from https://network.nvidia.com/products/infiniband-drivers/linux/mlnx_ofed/
+
+With that in place, the container may be built with Podman as here:
+
+```shell
+
+podman build --volume $(readlink -f rockylinux-mofed):/mnt:ro --file rockylinux-mofed/Containerfile rockylinux-mofed/
```

Comment:
I think this command needs to be run in the directory above. Alternatively, if the user is running in this directory. command would be something like this?
```
podman build --volume $(pwd):/mnt:ro --file Containerfile . 
```

Reply:
Yeah, that's fair. I tend to run podman from outside the directory so that it's more explicit on the command-line; but since README is in this directory, it makes sense to just use `.`.

I'll make that update. Thanks for the feedback!